pkg/products/rhsso/reconciler.go

```diff
@@ -215,105 +316,217 @@ func (r *Reconciler) reconcileTemplates(ctx context.Context, installation *integ
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
 
+// workaround: the keycloak operator creates a route with TLS passthrough config
+// this should use the same valid certs as the cluster itself but for some reason the
+// signing operator gives out self signed certs
+// to circumvent this we create another keycloak route with edge termination
```

Comment:
Is this going to be addressed in a future release of keycloak / the keycloak-operator? If so, can we create a Jira to remove this workaround at that point, so we don't forget?

Reply:
Yes, the PR to update the route in the keycloak operator has now been merged, but this is not in the 8.0.1 release: https://github.com/keycloak/keycloak-operator/pull/115